libs/ett/auth/feature-login/src/lib/ett-authentication-link.component.ts

```diff
@@ -45,26 +47,27 @@ const selector = 'ett-authentication-link';
   ],
   template: `
     <a *rxLet="loginUrl$ as loginUrl; rxError: loginError" [href]="loginUrl"
-      >NemID or MitID</a
+      >NemID</a
     >
     <ng-template #loginError let-error="$error">
       <p class="${selector}__error">
-        NemID and MitID login are currently unavailable. Please try again later.
+        NemID login is currently unavailable. Please try again later.
       </p>
     </ng-template>
   `,
 })
 export class EttAuthenticationLinkComponent {
-  #returnUrl = this.urlGenerator.fromCommands([ettDashboardRoutePath]);
+  #absoluteReturnUrl = this.urlGenerator.fromCommands([ettDashboardRoutePath]);
 
   loginProviderError$: Observable<unknown>;
```

Comment:
question:

Do we know the type of error to the Observable above?

Reply:
Which observable?